archive/gzip/gzip_test.go

```diff
@@ -44,7 +54,7 @@ func TestCreate(t *testing.T) {
 			name: "non-existing mount paths",
 			tgz:  New(log.NewNopLogger(), testRootMounted, true, flate.DefaultCompression),
 			srcs: []string{
-				"iamnotexists",
+				"idonotexist",
```

Comment:
🙈 It supposes to be a joke

Reply:
Idk why I was so serious changing this :laughing:, let me keep the joke then.